src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/EntityDetail/detailTextField.html.twig

```diff
@@ -5,7 +5,6 @@
             <i class="fa fa-question-circle"></i>
         </span>
     {% endif %}
-    <label>{{ label }}</label>
-    <span>{{ value }}</span>
+    <label>{{ label }}</label><span>{{ value }}</span>
```

Comment:
I personally prefer using 'spaceless', keeping the Twig markup slightly more readable.

https://twig.symfony.com/doc/2.x/tags/spaceless.html

Reply:
To me the solution looks equal, and if there were more fields I would certainly go for a spaceless filter, but for now removing the newline seems sufficient enough.

Be aware you're referring to a deprecated function, see:
https://twig.symfony.com/doc/2.x/filters/spaceless.html